documentation/sphinx/source/release-notes/release-notes-730.rst

```diff
@@ -2,6 +2,16 @@
 Release Notes
 #############
 
+7.3.27
+======
+* Same as 7.3.26 release with AVX disabled.
+
+7.3.26
+======
+* Fixed exclude command in fdbcli, allows excluding servers by locality.
```

Comment:
Can you add PRs?

Reply:
The change is mostly on the server side. So I'd rephrase as "Fixed several locality-based exclusion bugs."

- https://github.com/apple/foundationdb/pull/11024
- https://github.com/apple/foundationdb/pull/11007
- https://github.com/apple/foundationdb/pull/11005